### DISCUSSION TITLE
multiple triggers on same scaledObject

### DISCUSSION THREAD

**Discussion by yuvalweber** [kedacore/keda#4033] [Q&A / Need Help]
I was wondering if there is a possibility to get information from couple of datasources like prometheus, new relic, cpu and based on the result of all of those will scale or not.
Like for example if cpu and prometheus query is good then scale.
Is there option like this?

**Comment by tomkerkhove** [comment]
You can use multiple triggers today, however, do note that it will start scaling as soon as 1 of them meets the criteria

**Reply by yuvalweber** [reply]
is there a way to create like some kind of dependency graph?
Like if both condition are met or if something triggers and then another one triggers?

**Reply by zroubalik** [reply]
https://github.com/kedacore/keda/issues/3567

**Comment by gauravakg10** [comment]
@zroubalik @tomkerkhove Thanks for the info, Can we use multiple newrelic `nrql` under a single trigger type?
